vendor/assets/stylesheets/dvl/components/page_header.scss

```diff
@@ -0,0 +1,131 @@
+.page_header {
+  @include cf();
+  border-bottom: 1px solid $lightGray;
+  padding-bottom: 0.6rem;
+  margin-bottom: 1.25rem;
+  position: relative;
+  h2,
+  h3 {
+    margin-bottom: 0;
+    margin-right: 2rem;
+  }
+  h2 {
+    float: left;
+  }
+  h3 {
+    font-weight: 500;
+    color: $darkGray;
+    font-size: 1.3rem;
+    margin-top: 0.3rem;
+  }
+  .secondary_nav,
+  .tertiary_nav {
+    margin-top: 0.5rem;
+  }
+  .secondary_nav {
+    float: left;
+  }
+  .tertiary_nav {
+    float: right;
+  }
+}
+
+// Secondary and tertiary navigation, within page headers.
+
+.secondary_nav,
+.tertiary_nav {
+  font-weight: 600;
+  li {
+    display: inline;
+    > span {
+      color: $darkGray;
+    }
+    &.active a {
+      color: $darkestGray;
+      &:hover {
+        text-decoration: none;
+      }
+    }
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.secondary_nav {
+  font-size: 0.9rem;
+  li {
+    margin-right: 1.5rem;
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.tertiary_nav {
+  font-size: 0.8rem;
+  line-height: 1.8;
+  float: right;
+  li {
+    margin-right: 1rem;
+    a {
+      color: $darkerGray;
+
+      // Add a space in between icon & text
+      > i {
+        margin-right: 0.2rem;
+      }
+    }
+    &.active a {
+      color: $black;
+    }
+  }
+}
+
+// Subheaders on a page.
+
+.page_subheader {
```

Comment:
The rule is one "component" per file. This is also referred to as a "block" in BEM: https://en.bem.info/method/definitions/